requests/packages/idna/codec.py

```diff
@@ -0,0 +1,118 @@
+from idna.core import encode, decode, alabel, ulabel, IDNAError
```

Comment:
Hrm, ideally this will be an automated rewrite. @kjd as a note for the future, relative imports might make life a bit easier here. ;)

In this case this should be simple enough I think: `sed "s/from idna/from /"` for each file in the `idna` directory should do the trick.